### ISSUE TITLE
Please share the code

### ISSUE THREAD

**Issue by ZhengDanYang1** [tianzhiliang/RAM4CbR#1]
Hello, want to learn your code, please share your code.

**Comment by Arij-Aladel** [comment]
yeah t is strange to write on the article Aforementioned implementation details can
be found in our codes 1. and not find code when following the link
